pkgs/development/ocaml-modules/cduce/common.nix

```diff
@@ -0,0 +1,14 @@
+{ lib }:
+
+{
+  version = "0.7.0-rc11";
+  useDune2 = true;
```

Comment:
The `useDune2` attribute is deprecated.

Reply:
My bad I read this line wrong and believed it was still in use:
https://github.com/NixOS/nixpkgs/blob/350fd0044447ae8712392c6b212a18bdf2433e71/pkgs/build-support/ocaml/dune.nix#L6

Reply:
Do not hesitate to suggest improvements to the documentation (https://nixos.org/manual/nixpkgs/unstable/#sec-language-ocaml-packaging).

Reply:
I think I will just do a tree-wide rewrite to remove useDune2 and add a warning. I will need to build all packages before to see any regression.